doc/user-manual/language/cubical.lagda.rst

```diff
@@ -107,8 +113,8 @@ the following to the top of a file (this assumes that the
   open import Cubical.Core.Everything
 
 For detailed install instructions for ``agda/cubical`` see:
-https://github.com/agda/cubical/blob/master/INSTALL.md. In order to
-make this library visible to Agda add
+https://github.com/agda/cubical/blob/4de6b6939245ce281f02e47af02f2deb1cbd853e/INSTALL.md. In
+order to make this library visible to Agda add
```

Comment:
I don't think this is better - if we link to specific commit, we should keep this link updated. Shouldn't we just say that people should follow the instructions at https://github.com/agda/cubical to install cubical?

Reply:
```suggestion
In order to make this library visible to Agda add
```

Reply:
Thanks, good point! Fixed it the way you suggested and deleted the link